### PR TITLE
Option descriptions

### PR DESCRIPTION
**Pull request by cgeers**
## Pull Request Checklist

**Is this in reference to an existing issue?**
#13 
#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: bin/check-log.rb [hunk 1]
```diff
@@ -85,13 +86,13 @@ class CheckLog < Sensu::Plugin::Check::CLI
          long: '--encoding ENCODING-PAGE'
 
   option :warn,
-         description: 'Warning level if pattern has a group',
+         description: 'Warning level if pattern is found >= N times',
```
Comment (majormoses) [inline bin/check-log.rb]: If this is true then we should probably make it a default of `1` as no default will eval to `nil` when calling `.to_i` it becomes `0` so it should either be a default of 1 or remove the greater equality for a greater than.
```
$ irb
irb(main):001:0> nil.to_i
=> 0
```

File: bin/check-log.rb [hunk 2]
```diff
@@ -85,13 +86,13 @@ class CheckLog < Sensu::Plugin::Check::CLI
...
          short: '-w N',
          long: '--warn N',
          proc: proc(&:to_i)
 
   option :crit,
-         description: 'Critical level if pattern has a group',
+         description: 'Critical level if pattern is found >= N times',
```
Comment (majormoses) [inline bin/check-log.rb]: If this is true then we should probably make it a default of `1` as no default will eval to `nil` when calling `.to_i` it becomes `0` so it should either be a default of 1 or remove the greater equality for a greater than.
```
$ irb
irb(main):001:0> nil.to_i
=> 0
```

**Review by majormoses — CHANGES_REQUESTED** [review]
Thanks for finding and fixing this, I think we need one more tweak before calling this a solved issue.

**Comment by majormoses** [comment]
Can you please include and automated or manual [testing artifact](https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md#7-testing-artifacts)?

**Comment by cgeers** [comment]
I opted to revert the original behavior of defaulting to the critical severity if any matches are found. 

I've included an integration test covering basic behavior of the check-log.rb check.

**Review by majormoses — APPROVED** [review]

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-logs/versions/1.3.2

**Pull request merged**